src/components/miscellaneous/ShareButton.tsx

```diff
@@ -0,0 +1,40 @@
+// Material UI Components
+import Button from '@material-ui/core/Button';
+
+import useShare from 'use-share';
+
+export type ShareProps = {
```

Comment:
Hvis du importerer ButtonProps og legge til i ShareProps med en `&` så vil man kunne legge til props som tilhører Button også til ShareButton og da legge de til Button med `{...props}` som i kommentaren lengre nede.

```suggestion
import Button, { ButtonProps } from '@material-ui/core/Button';

import useShare from 'use-share';

export type ShareProps = ButtonProps & {
```